contrib/netavark/template.py

```diff
@@ -10,3 +10,11 @@
 url = "https://github.com/containers/netavark"
 source = f"{url}/archive/v{pkgver}.tar.gz"
 sha256 = "b1422ef6927458e9f80f7d322b751e29ab5d04d8ed6cb065baa82fa4291af10f"
+
+
+def do_install(self):
+    self.install_file(
```

Comment:
install_bin

Reply:
install_bin only goes to usr/bin tho which is not where this is going

Reply:
oh right, q66 moment